### PR TITLE
MS Graph API ingester

### PR DESCRIPTION
**Pull request by floren**

### CODE REVIEW

File: ingesters/MSGraphIngester/main.go [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+		lg.Fatal("Failed build our ingest system: %v", err)
+	}
+	defer igst.Close()
+	debugout("Starting ingester muxer\n")
+	if err := igst.Start(); err != nil {
+		lg.FatalCode(0, "Failed start our ingest system: %v", err)
+		return
+	}
+
+	debugout("Waiting for connections to indexers ... ")
```
Comment (djfritz) [inline ingesters/MSGraphIngester/main.go:148]: missing newline?

**Review by djfritz — CHANGES_REQUESTED** [review]

File: ingesters/MSGraphIngester/main.go (hunk 1 above)
Comment (floren) [reply ingesters/MSGraphIngester/main.go:148]: The intent is that it'll show "Wait for connections to indexers ... Successfully connected to ingesters"

Comment (djfritz) [reply ingesters/MSGraphIngester/main.go:148]: ok

**Review by djfritz — APPROVED** [review]
LGTM

**Pull request merged**